packages/gatsby/src/utils/api-runner-node.js

```diff
@@ -98,6 +106,7 @@ const runAPI = (plugin, api, args) => {
       {
         ...args,
         pathPrefix,
+        assetPrefix,
```

Comment:
This is provided merely for convenience; I think the `withAssetPrefix` and `withPathPrefix` should be used instead.